File: Day4/Python/Ques15.py
import math
n=(int(input("Enter a number:")))
a=b=n
count=t=0
while n>0:
    n=n//10
    count +=1
while a>0:
    t=t+ math.pow(a%10,count)
    a=a//10
if t==b:
    print("Given number is Armstrong No")
else:
    print("Given number is not a Armstrong No")
    

File: Day4/C/Ques15.c
#include<stdio.h>
int main(){
    int n,count=0,temp,sum=0;
    printf("Enter a number:");
    scanf("%d",&n);
    int a=n,b=n;
    while(n>0){
        count+=1;
        n=n/10;
    }
    while(a>0){
        temp=(a%10);
        int t=1;
        for(int i=0;i<count;i++){
            t=t*temp;
        }
        sum=sum+t;
        a=a/10;
    }
    if(sum==b){
        printf("%d is an armstrong number",b);
    }
    else{
        printf("%d is not a armstrong number",b);
    }
    return 0;
}